Automattic/wp-calypso
# Sites Management Page: optimise banner that promotes tailored flows

Fixes https://github.com/Automattic/wp-calypso/issues/68760

#### Proposed Changes

- [x] Optimise images, so they load faster on slow networks. See [here](https://github.com/Automattic/wp-calypso/pull/68483#pullrequestreview-1130930739). 
- [x] Change banner type according to available space: double-tile should become tile. See [here](https://github.com/Automattic/wp-calypso/pull/68483#issuecomment-1268852466).
- [x] Don't show the banner when it wraps onto a line by itself. See [here](https://github.com/Automattic/wp-calypso/pull/68483#pullrequestreview-1130930739). 

#### Testing Instructions



Note: I've included the temporary URL param again for this PR: you can control the site count with `?sitecount=<number>`

* navigates to `/sites` without a Link in Bio site as one of your sites
* You can simulate slow networks with Chrome. I set up a 4G profile and used that. (see [comment](https://github.com/Automattic/wp-calypso/pull/68805#issuecomment-1272912184))
* Try resizing your browser and verify that 
   - the `double-tile` banner is converted to a `tile` banner below `960px`
   - if there are two sites and the screen width is below `960px` the banner is no longer rendered to save it wrapping
* Create a Link in Bio site and retest
* check for regressions 

#### Pre-merge Checklist






Related to #
